applications/nrf5340_audio/src/modules/lc3_file.c

```diff
@@ -12,6 +12,34 @@ LOG_MODULE_REGISTER(sd_card_lc3_file, CONFIG_MODULE_SD_CARD_LC3_FILE_LOG_LEVEL);
 
 #define LC3_FILE_ID 0xCC1C
 
+static void lc3_header_print(struct lc3_file_header *header)
+{
+	if (header == NULL) {
+		LOG_ERR("Nullptr received");
+		return;
+	}
+
+	LOG_DBG("File ID: 0x%04x", header->file_id);
+	LOG_DBG("Header size: %d", header->hdr_size);
+	LOG_DBG("Sample rate: %d Hz", header->sample_rate * 100);
+	LOG_DBG("Bit rate: %d bps", header->bit_rate * 100);
+	LOG_DBG("Channels: %d", header->channels);
+	LOG_DBG("Frame duration: %d us", header->frame_duration * 10);
+	LOG_DBG("Num samples: %d", header->signal_len_lsb | (header->signal_len_msb << 16));
+}
+
+int lc3_header_get(struct lc3_file_ctx const *const file, struct lc3_file_header *header)
+{
+	if ((file == NULL) || (header == NULL)) {
+		LOG_ERR("Nullptr received");
```

Comment:
Nitpick but not important: since in the later part I see we use "null pointer" instead of "nullptr"
We can update in here too, but not very necessary.

Reply:
It seems that in this file it is Nullptr all places. The file was originally written by Andreas, so I continued in the same style